lxd/device/nic_bridged.go

```diff
@@ -129,6 +139,16 @@ func (d *nicBridged) validateConfig(instConf instance.ConfigReader) error {
 			if dhcpv6Subnet != nil && !dhcpalloc.DHCPValidIP(dhcpv6Subnet, nil, net.ParseIP(d.config["ipv6.address"])) {
 				return fmt.Errorf("Device IP address %q not within network %q subnet", d.config["ipv6.address"], n.Name())
 			}
+
+			ip, _, err := net.ParseCIDR(netConfig["ipv6.address"])
```

Comment:
Same question around handling of `ipv6.address` being empty or being set to `none`.